package.json

```diff
@@ -2,6 +2,10 @@
    "name": "quick-cashu",
    "version": "0.1.0",
    "private": true,
+   "engines": {
+      "node": "20.x",
```

Comment:
this now means that vercel will always build with latest node version 20 and locally you will be forced to use any version 20. however if you use nvm it will still use 20.18.0 as specified in nvmrc. that way at least dev locally can be in sync